picker-starter/src/options-schema.ts

```diff
@@ -20,8 +20,3 @@ const optionsSchemaMixin: Joi.StrictSchemaMap<MockPluginOptions> = {
 export const optionsSchema = Joi.object<MockPluginOptions, true>({
   ...optionsSchemaMixin,
 }).unknown(true)
-
-export const optionsExample: MockPluginOptions = {
```

Comment:
This example was moved to the `config.picker.ts`.

We may get rid of all this `option-schema.ts` file and put the validation rules directly into the `picker.config.ts`.